src/components/views/elements/MiniAvatarUploader.tsx

```diff
@@ -82,11 +88,13 @@ const MiniAvatarUploader: React.FC<IProps> = ({ hasAvatar, hasAvatarLabel, noAva
         >
             { children }
 
-            <Tooltip
-                label={label}
-                visible={!!label && (hover || show)}
-                forceOnRight
-            />
+            <div className={classNames("mx_Tooltip", {
+                "mx_Tooltip_visible": visible,
+                "mx_Tooltip_invisible": !visible,
+            })}>
```

Comment:
it's a bit of a smell if we're ripping classes out of a component just to use them here. Can we make it a render mode of the `Tooltip` to do the work for us?

Reply:
We could but it'd make it an even more confusing component and we need to rebuild out Tooltips altogether as it is

Reply:
I'm a bit torn because this is the sort of thing that we should probably avoid making worse, but the situation is so bad that it's on a shortlist of options.

@jryans might be best to review this

Reply:
The issues are quite annoying visually, so I think it's better to accept this change and then revisit tooltips holistically later.